src/realm/mixed.hpp

```diff
@@ -301,6 +304,18 @@ inline Mixed::Mixed(Timestamp v) noexcept
     }
 }
 
+inline Mixed::Mixed(Decimal128 v) noexcept
+{
+    if (!v.is_null()) {
+        m_type = type_Decimal + 1;
+        auto x = v.to_bid64();
```

Comment:
Are we ok with this imprecise representation when using Mixed? Shouldn't we expand the Mixed storage to 128 bits?

Reply:
Probably not in the long run. We will have to figure out what we can live with. I would hate if the Mixed type gets bigger that 16 bytes, but maybe there is no way around it.

Reply:
I don't see any good alternative. Probably good to make the change early if it's breaking. Otherwise we should probably try-catch the `to_bid64()` conversion, because it'll throw if the conversion fails, and this is a noexcept method.

Reply:
I will add a FIXME.
Be aware that this functionality is mostly used by the sorting functionality. It will only be in corner cases this lack of precision matters. And be aware that the scope of this PR is only to provide initial support so that the SDK's can get going.